dev-aws/kafka-shared-msk/cbc/cbc.tf

```diff
@@ -16,6 +16,23 @@ resource "kafka_topic" "FraudEvents" {
   }
 }
 
+resource "kafka_topic" "data_product_events_v1" {
+  name = "cbc.DataProductEvents"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
```

Comment:
to be honest, this one is a new topic and it's for the data infra integration. Maybe it could do with less than infinite. I default to infinite because I'm concerned we'll later find out that actually we need to reconsume everything for whatever reason, but maybe there's a saner default for this topic specifically, considering it's specifically for this data infra use-case.

Reply:
Is it possible to find out for how long is UW required to keep this data? 
We could also go with infinite and revisit later, but it would be good to settle this from the get go.

Reply:
the data will be kept in braze anyways, so how long it lives in the topic will not necessarily match that I suppose. I've changed it to a month so that if something breaks we have plenty of time to fix it without risk of losing messages. I've asked data infra whether they have any recommendations for this value though, so won't merge just yet, but I guess a month is a saner value than forever for this topic.

Reply:
I haven't received an answer from data-infra yet, but given this is dev I'm happy to just stick with 1 month for it. We won't even have much data going through this one in dev anyways.